packages/amplify-provider-awscloudformation/lib/delete-env.js

```diff
@@ -1,9 +1,10 @@
 const Cloudformation = require('../src/aws-utils/aws-cfn');
-
+const { loadConfigurationForEnv } = require('./configuration-manager');
 
 async function run(context, envName) {
-  return new Cloudformation(context)
-    .then(cfnItem => cfnItem.deleteResourceStack(envName));
+  const credentials = await loadConfigurationForEnv(context, envName);
```

Comment:
this does not actually return the credentials associated with the specific env.

Reply:
We do return the config for env now